R/date_cut.R

```diff
@@ -58,6 +60,15 @@ date_cut <- function(dataset_sdtm,
   assert_data_frame(dataset_cut,
     required_vars = vars(USUBJID, !!cut_var)
   )
+  assert_that(
+    (length(get_duplicates(dataset_cut$USUBJID)) == 0),
+    msg = "Duplicate patients in the DCUT (dataset_cut) dataset, please update."
+  )
+  assert_that(
+    (any(is.na(mutate(dataset_cut, !!cut_var))) == FALSE),
+    msg = "At least one patient with missing DCUTDTM in the DCUT
```

Comment:
```suggestion
    msg = "At least one patient with missing datacut date (cut_var) in the DCUT
```
Suggest updating DCUTDTM reference in warning also